docs/Contribute-to-AlmaLinux-Build-System.md

```diff
@@ -0,0 +1,212 @@
+---
+title: 'AlmaLinux Build System'
+---
+
+###### last updated: 2024-09-20
+
+# Contribute to AlmaLinux Build System
+
+The AlmaLinux Build System is managed by the [SIG/Build System](/sigs/Build-System) SIG. 
+
+The project is open for community contributions. If you are interested in contributing to the Build System project, we recommend you start by checking the AlmaLinux Build System project [milestones](https://github.com/AlmaLinux/build-system/milestones) for ongoing tasks and following the guidelines below.
+
+When you are ready to create a pull request with your suggested changes, please follow the **[commit guidelines](#commit-guidelines)** to keep our commit messages consistent across all repositories around the AlmaLinux Build System. 
+
+## Contribute to the codebase
+
+To contribute to development we recommend deploying the AlmaLinux Build System [locally](https://github.com/AlmaLinux/build-system/wiki/Deploy-on-local-and-remote-machines) and following these guide steps for possible modifications and commit guidelines.
+
+### Help Wanted
```

Comment:
Hey @sboldyreva, I actually talked to the team and we compiled the following list:
* Help on keeping our Build System SIG documentation up to date (Readmes in repos, wiki, docs, sig wiki page, etc)
* Testing (experience with pytest), we need help to:
  * Increase our test coverage in repos that already have tests
  * Add tests to those that don't have tests at all
  * Design and implement of integration/e2e tests that involve different services
* Famliar with Ansible? Help us in testing and improving our current ansible roles to deploy ALBS
* Interested in supply-chain security and SBOM? Help us in defining next steps towards providing and expanding the current SBOM data that AlmaLinux is generating

Also, I think that we can provide a short list of our tech stack, something like:
* Backend: Python, FastAPI, SQLAlchemy, PostgreSQL, redis
* Frontend: JavaScript, Vue.js, Quasar
* Tooling: Docker, Docker Compose, Ansible, Terraform

What do you think?

Reply:
Love this description here.

Reply:
@bennyvasquez @javihernandez added